Use CXXFLAGS_FOR_BUILD and LDFLAGS_FOR_BUILD for the tools since expat
may not be living in the default include & library path.

Signed-off-by: Gustavo Zacarias <gustavo@zacarias.com.ar>

diff -Nura libdbus-c++-0.9.0.orig/configure.ac libdbus-c++-0.9.0/configure.ac
--- libdbus-c++-0.9.0.orig/configure.ac	2014-01-05 15:28:03.653886567 -0300
+++ libdbus-c++-0.9.0/configure.ac	2014-01-05 15:44:32.571172225 -0300
@@ -65,7 +65,11 @@
 AC_PROG_CXX
 
 CXX_FOR_BUILD=${CXX_FOR_BUILD-${CXX}}
+CXXFLAGS_FOR_BUILD=${CXXFLAGS_FOR_BUILD-${CXXFLAGS}}
+LDFLAGS_FOR_BUILD=${LDFLAGS_FOR_BUILD-${LDFLAGS}}
 AC_SUBST(CXX_FOR_BUILD)
+AC_SUBST(CXXFLAGS_FOR_BUILD)
+AC_SUBST(LDFLAGS_FOR_BUILD)
 
 AM_PROG_LIBTOOL
 
diff -Nura libdbus-c++-0.9.0.orig/tools/Makefile.am libdbus-c++-0.9.0/tools/Makefile.am
--- libdbus-c++-0.9.0.orig/tools/Makefile.am	2014-01-05 15:28:03.652886535 -0300
+++ libdbus-c++-0.9.0/tools/Makefile.am	2014-01-05 15:44:42.071482390 -0300
@@ -1,6 +1,8 @@
 # hacky, but ...
 
 CXX = $(CXX_FOR_BUILD)
+CXXFLAGS = $(CXXFLAGS_FOR_BUILD)
+LDFLAGS = $(LDFLAGS_FOR_BUILD)
 
 AM_CPPFLAGS = \
 	$(dbus_CFLAGS) \
